xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -363,59 +336,33 @@ boolean isDeactivated() {
       }
 
       class ChildHelper extends ForwardingLoadBalancerHelper {
-
-        private final OrcaReportingHelperWrapper orcaReportingHelperWrapper;
         private SubchannelPicker currentChildPicker = XdsSubchannelPickers.BUFFER_PICKER;
         private ConnectivityState currentChildState = CONNECTING;
 
-        ChildHelper() {
-          final ClientLoadCounter counter = loadStatsStore.getLocalityCounter(locality);
```

Comment:
This was potentially a problem. The lifecycle of a `ChildHelper` is different from the `LoadStatsStore` that used to record loads for the cluster that this locality belongs to. A `ChildHelper`'s lifecycle is same as the presence of the corresponding locality. However, a `LoadStatsStore`'s lifecycle is for the time since load reporting was last enabled. They are completely disjoint, so a `ChildHelper` should always pick up the freshest `LoadStatsStore` to record load.

Reply:
Obsolete.